### PR TITLE
Kontena Cloud nodes

### PR DESCRIPTION
**Pull request by jakolehm**

### CODE REVIEW

File: using-kontena/install-nodes/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Almost any machine may be turned into a Kontena Node (see [system requirements](#system-requirements)).
 
-The easiest way for adding Kontena Nodes is to [use the provisioning tool](#adding-kontena-nodes-with-provision-tool) that is built into Kontena CLI. This tool may be used to provision Kontena Nodes to most major public cloud platforms and for local testing purposes.
+The easiest way for adding Kontena Nodes is to [use the provisioning tool](#adding-kontena-nodes-with-provision-tool) that is built into Kontena CLI. This tool may be used to provision Kontena Nodes to Kontena Cloud, most major public cloud platforms and for local testing purposes.
```
Comment (saana-kontenacm) [inline using-kontena/install-nodes/README.md]: The easiest way to add Kontena Nodes is to [use the provisioning tool](#adding-kontena-nodes-with-provision-tool) that is built into the Kontena CLI.

File: using-kontena/install-nodes/cloud.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+```
+$ kontena cloud node terminate <NAME>
+```
+
+## Update a Kontena Node
+
+
+### Kontena Agent
+
+By default, Kontena Nodes will auto-update to the same version as Kontena Platform Master immediately if the `major` or `minor` version changes (`major.minor.x`). Kontena Nodes will do a patch level auto-update only when rebooted or systemd is restarted. This auto-update process can be forced by executing the following command in every Kontena Node host:
```
Comment (saana-kontenacm) [inline using-kontena/install-nodes/cloud.md]: By default, Kontena Nodes will auto-update to the same version as the Kontena Platform Master immediately if the `major` or `minor` version changes (`major.minor.x`). Kontena Nodes will do a patch level auto-update only when rebooted or systemd is restarted. This auto-update process can be forced by executing the following command in every Kontena Node host:

File: using-kontena/install-nodes/cloud.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+```
+$ kontena cloud node shell <NAME> systemctl restart kontena-agent
+```
+
+### Host Operating System
+
+Kontena Nodes use [Container Linux](https://coreos.com/os/docs/latest/) as the host operating system. Host OS is configured to download updates automatically but they are applied only when a Kontena Node is rebooted. To apply updates, reboot a node with following command:
```
Comment (saana-kontenacm) [inline using-kontena/install-nodes/cloud.md]: Kontena Nodes use [Container Linux](https://coreos.com/os/docs/latest/) as the host operating system. Host OS is configured to download updates automatically but they are applied only when a Kontena Node is rebooted. To apply updates, reboot a node with the following command:

File: using-kontena/install-nodes/cloud.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+* [Prerequisites](#prerequisites)
+* [Create Kontena Node](#install-kontena-node)
+* [Restart Kontena Node](#restart-kontena-node)
+* [Terminate Kontena Node](#terminate-kontena-node)
+* [Update Kontena Node](#update-kontena-node)
+* [Advanced Usage](#advanced-usage)
+
+## Prerequisites
+
+* [Kontena CLI](/tools/cli.md) with the latest version of `cloud` plugin. If you don't have the plugin installed, you can install it with the: `kontena plugin install cloud` command.
```
Comment (nevalla) [inline using-kontena/install-nodes/cloud.md:12]: Should we also mention how to update plugin?
```
If you don't have the plugin installed or you want to update the plugin, 
you can do it with the: `kontena plugin install cloud` command.
```

**Pull request merged**